### ISSUE TITLE
[Question/Feedback]: Deploy AMBA Notification Assets policy default parameter for ALZMonitorActionGroupEmail cannot be changed.

### ISSUE THREAD

**Issue by mwmeehan** [Azure/azure-monitor-baseline-alerts#130]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Description

We are testing an assignment for AMBA and noticed that all the parameters we changed work fine and show user defined parameter except ALZMonitorActionGroupEmail which stays at default, even though we changed that as well.  Code we have for it is below;

"policyAssignmentParametersServiceHealth": {
      "value": {
        "ALZMonitorActionGroupEmail": {
          "value": "emailishere@bug.com"
        },

**Comment by Brunoga-MS** [comment]
Hello @mwmeehan ,
thanks for your feedback. Looking at the code we have in the branch now, the email parameters is not located under the **_policyAssignmentParametersServiceHealth_** section anymore. It is instead located under the **_policyAssignmentParametersNotificationAssets_** section. Could you please verify that the parameter file you are using is the correct one?

**Comment by mwmeehan** [comment]
> Hello @mwmeehan , thanks for your feedback. Looking at the code we have in the branch now, the email parameters is not located under the **_policyAssignmentParametersServiceHealth_** section anymore. It is instead located under the **_policyAssignmentParametersNotificationAssets_** section. Could you please verify that the parameter file you are using is the correct one?

I am looking into this now with my team.  Thank you for the quick response!

**Comment by Brunoga-MS** [comment]
Hello @mwmeehan ,
any news on this one?

**Comment by mwmeehan** [comment]
> Hello @mwmeehan , any news on this one?
> 
> Thanks, Bruno.

Yes, we looked at it this week and it looks like we were using an old version.  We have updated it and it applies correctly now.  I will close this issue.  Appreciate the assistance!
